AirCasting/CreateSessionViews/ConnectingABViewModel.swift

```diff
@@ -0,0 +1,29 @@
+// Created by Lunar on 21/07/2021.
+//
+
+import CoreBluetooth
+import Foundation
+
+protocol ConnectingABProtocol {
+    var bluetoothManager: BluetoothManager { get }
+    func performConnectingWithin10Second(peripheral: CBPeripheral, completion: @escaping () -> Void)
+}
+
+class ConnectingABViewModel: ConnectingABProtocol {
+    var bluetoothManager: BluetoothManager
```

Comment:
see - now ViewModel is mixed with Bluetooth stuff and everything is connected with each other, no chance to test in isolation and any business requirement change would require everything to be changed. Not too great.